### ISSUE TITLE
Error when using a list of paths in Image class

### ISSUE THREAD

**Issue by dmus** [fepegar/torchio#762]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug summary

See reproducible example, it gives an RuntimeError: Exception thrown in SimpleITK ImageFileReader_ReadImageInformation

### Code for reproduction

```python
path = '/tmp/tmpy168nb8s/Results/test_cvf/20211129_180113_860_3QC1RFRH/saved_images/prediction_test_cvf_001.mhd'
prediction = tio.Image(path=[path])

print(prediction.affine)
```

### Actual outcome

I would expect that this works without problems and gives a similar outcome when passing a single path instead of a list as the `path` argument

### Error messages

```python-traceback
File "reproduce.py", line 27, in <module>
    print(prediction.affine)
  File "/home/derk/rndeep/venv/lib/python3.6/site-packages/torchio/data/image.py", line 237, in affine
    affine = read_affine(self.path)
  File "/home/derk/rndeep/venv/lib/python3.6/site-packages/torchio/data/io.py", line 106, in read_affine
    reader = get_reader(path)
  File "/home/derk/rndeep/venv/lib/python3.6/site-packages/torchio/data/io.py", line 115, in get_reader
    reader.ReadImageInformation()
  File "/home/derk/rndeep/venv/lib/python3.6/site-packages/SimpleITK/SimpleITK.py", line 8673, in ReadImageInformation
    return _SimpleITK.ImageFileReader_ReadImageInformation(self)
```

### Expected outcome

I would expect it print the affine

### System info

```shell
Platform:   Linux-5.4.0-62-generic-x86_64-with-Ubuntu-18.04-bionic
TorchIO:    0.18.69
PyTorch:    1.10.0+cu102
SimpleITK:  1.2.4 (ITK 4.13)
NumPy:      1.19.5
Python:     3.6.9 (default, Jan 26 2021, 15:33:00) 
[GCC 8.4.0]
```

**Comment by dmus** [comment]
Maybe my message was not very clear, it is about the difference between

```
prediction = tio.Image(path=path)

print(prediction.affine)
```
and
```
prediction = tio.Image(path=[path])

print(prediction.affine)
```
The first works and the second not

**Comment by fepegar** [comment]
Oh, I see. Thanks. I will look into it.

For now, you can call `prediction.load()` before getting `prediction.affine` to avoid this error.

**Comment by fepegar** [comment]
Fixed in `v0.18.70`. Thanks for reporting, @dmus.
